packages/commonwealth/client/scripts/controllers/app/webWallets/keplr_ethereum_web_wallet.ts

```diff
@@ -167,6 +167,7 @@ class EVMKeplrWebWalletController implements IWebWallet<AccountData> {
       this._accounts = await this._offlineSigner.getAccounts();
       this._enabled = true;
       this._enabling = false;
+      localStorage.setItem('wallet', WalletId.Keplr); // so submitTx uses the correct wallet
```

Comment:
Can we use the saved wallet_id on the Address object?

Reply:
Yep. That's better. Done.